features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Resources/components/operationComponents/OperationGovernance.jsx

```diff
@@ -46,6 +48,16 @@ export default function OperationGovernance(props) {
         operation, operationsDispatcher, operationRateLimits, api, disableUpdate, spec, target, verb,
     } = props;
     const isOperationRateLimiting = api.apiThrottlingPolicy === null;
+
+    /**
+     *
+     *
+     * Remove scope
+     */
+    function removeScope() {
+        operationsDispatcher({ action: 'scopes', data: { target, verb, value: '' } });
```

Comment:
Shall we set the value as an empty array here ?
```suggestion
        operationsDispatcher({ action: 'scopes', data: { target, verb, value: [] } });
```

so that we don't want to check for empty string in resource dispatcher?

Reply:
And also we could call this dispatcher in `onClick` handler with inline lambda function

```javascript
<IconButton onClick={()=> operationsDispatcher({ action: 'scopes', data: { target, verb, value: [] } })} aria-label='delete'>
```